frame/nfts/src/lib.rs

```diff
@@ -1257,11 +1323,55 @@ pub mod pallet {
 			collection: T::CollectionId,
 			maybe_item: Option<T::ItemId>,
 			key: BoundedVec<u8, T::KeyLimit>,
+			namespace: AttributeNamespace<T::AccountId>,
 		) -> DispatchResult {
 			let maybe_check_owner = T::ForceOrigin::try_origin(origin)
 				.map(|_| None)
 				.or_else(|origin| ensure_signed(origin).map(Some).map_err(DispatchError::from))?;
-			Self::do_clear_attribute(maybe_check_owner, collection, maybe_item, key)
+			Self::do_clear_attribute(maybe_check_owner, collection, maybe_item, key, namespace)
+		}
+
+		/// Approve item's attributes to be changed by a delegated third-party account.
+		///
+		/// Origin must be Signed and must be an owner of the `item`.
+		///
+		/// - `collection`: A collection of the item.
+		/// - `item`: The item that holds attributes.
+		/// - `delegate`: The account to delegate permission to change attributes of the item.
+		///
+		/// Emits `ItemAttributesApprovalAdded` on success.
```

Comment:
can approval of an item be signed by the owner for later use on chain by the delegated third-party? If so would be great to mention this in the docs here.